sdk/messaging/azeventhubs/README.md

```diff
@@ -98,36 +97,63 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
```

Comment:
The Contributing section is different from what is in this document: https://github.com/Azure/azure-sdk-for-go/pull/20382/commits/3a5bcc1c2cf8b19be181343952f13bc00d4497e9

Consider standardizing?